browser/ui/views/commands/accelerator_service.cc

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/views/commands/accelerator_service.h"
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "base/containers/contains.h"
+#include "base/containers/cxx20_erase_vector.h"
+#include "base/containers/flat_map.h"
+#include "brave/app/command_utils.h"
+#include "brave/components/commands/common/accelerator_parsing.h"
+#include "brave/components/commands/common/accelerator_pref_manager.h"
+#include "brave/components/commands/common/commands.mojom-forward.h"
```

Comment:
you don't need to include `mojom-forward.h` if you include `mojom.h`.

Reply:
Ah, my bad - I've got auto imports set up and it often drags in things it shoudn't.